File: src/main/java/snake/difficulty/Beginner.java
package snake.difficulty;

public class Beginner extends Difficulty {
    public Beginner() {
        this.delay = 200;
    }

    @Override
    public void changeDelay(int foodConsumed) {
        if (delay > 70 && foodConsumed % 6 == 0) {
            delay -= 10;
        }
    }
}

File: docs/difficulty/Beginner.js
import {Difficulty} from "./Difficulty.js";
export class Beginner extends Difficulty {
    constructor() {
        super(200);
    }
    changeDelay(foodConsumed) {
        if (this.delay > 70 && foodConsumed % 6 === 0) {
            this.delay -= 10;
        }
    }
}
